x-pack/plugins/alerting/server/alerts_client/lib/get_summarized_alerts_query.ts

```diff
@@ -340,11 +347,19 @@ export const getQueryByScopedQueries = ({
   };
 };
 
-const generateAlertsFilterDSL = (alertsFilter: AlertsFilter): QueryDslQueryContainer[] => {
+const generateAlertsFilterDSL = (
+  alertsFilter: AlertsFilter,
+  options?: { analyzeWildcard?: boolean }
+): QueryDslQueryContainer[] => {
   const filter: QueryDslQueryContainer[] = [];
+  const { analyzeWildcard = false } = options || {};
 
   if (alertsFilter.query) {
-    filter.push(JSON.parse(alertsFilter.query.dsl!));
+    const parsedQuery = JSON.parse(alertsFilter.query.dsl!);
+    if (analyzeWildcard) {
+      injectAnalyzeWildcard(parsedQuery);
```

Comment:
Is it possible to use `buildEsQuery` instead of having our own implementation?

Reply:
no

Reply:
😄. Thanks for the clarification.